KalturaWowzaServer/src/main/java/com/kaltura/media/server/wowza/CuePointsManager.java

```diff
@@ -321,62 +331,67 @@ protected void onAppStart(IApplicationInstance applicationInstance) {
 		applicationInstance.addLiveStreamPacketizerListener(liveStreamPacketizerListener);
 	}
 
-	protected void onPublish(IMediaStream stream, String entryId) {
+	protected void onPublish(final IMediaStream stream, final String entryId) {
 		if(stream.getClientId() < 0){
 			logger.debug("Stream [" + stream.getName() + "] entry [" + entryId + "] is a transcoded rendition");
 			return;
 		}
 
-		logger.debug("Stream [" + stream.getName() + "] entry [" + entryId + "]");
+		String streamName = stream.getName();
+		logger.debug("Stream [" + streamName + "] entry [" + entryId + "]");
 		ILiveStreamManager liveManager = KalturaServer.getManager(ILiveStreamManager.class);
```

Comment:
You have that member inherited from KalturaCuePointsManager (although I don't understand why it's there in the first place)